cmd/server/main.go

```diff
@@ -6,24 +6,42 @@ import (
 	"flag"
 	"fmt"
 	"log"
+	"net/http"
 	"os"
 )
 
 func main() {
 	addr := flag.String("addr", ":8080", "Server address")
-	elastic := flag.String("elastic", "http://elastic:9200", "ES address")
-	downloads := flag.String("downloads", "/tmp/downloads/", "Downloads folder path")
+	elastic := flag.String("elastic", "", "ES address")
 	index := flag.String("index", "files", "ES Index name")
+	sql := flag.String(
+		"sql",
+		"",
+		"Postgres connection string")
+	downloads := flag.String("downloads", "/tmp/downloads/", "Downloads folder path")
+
 	flag.Parse()
 
-	elasticClient, err := client.NewElasticClient(*elastic, *index)
-	if err != nil {
-		log.Fatal(err)
-	}
+	var server *http.Server
```

Comment:
please replace with 
var storageClient handler.Storage

and then if:
storageClient, err = client.NewElasticClient(*elastic, *index)
else 
storageClient, err = client.NewPostgresClient(*sql)

server := srv.NewFileServer(storageClient, *addr)